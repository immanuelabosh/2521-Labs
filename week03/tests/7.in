0
-8
1
10
22
11
5
-7
-10