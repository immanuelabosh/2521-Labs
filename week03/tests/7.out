Tree:
       0
      / \
     /   \
    -8    1
   / \     \
  /   \    10
-10   -7   / \
          5  22
             /
            11

#nodes = 9,  #leaves = 4
Infix  : -10 -8 -7 0 1 5 10 11 22 
Prefix : 0 -8 -10 -7 1 10 5 22 11 
Postfix: -10 -7 -8 5 11 22 10 1 0 
ByLevel: 0 -8 1 -10 -7 10 5 22 11 
